gatsby/howitworks.rst

```diff
@@ -42,4 +39,4 @@ everytime we create/edit/delete a file in the source-code.
 
```

Comment:
Sections 1 and 2 above need to follow the one line per sentence styleguide rule.

s/apis/APIs

Replace typographic quote marks around "props" with straight double quote marks `"`.

Gatsby or GatsbyJS?

"every time" is two words

"source code" is not hyphenated

Reply:
"source code" is not hyphenated